### ISSUE TITLE
XBB.1.9.2 with orf1a:T333M, Y2266C in China (7 seqs, 4 provinces)

### ISSUE THREAD

**Issue by aviczhl2** [cov-lineages/pango-designation#1769]
Sequences: EPI_ISL_17197742,EPI_ISL_17228016-17228017 （2023.2.24-2023.3.1）
No. of seqs: 3 (2 Liaoning, 1 Jiangsu）
Mutations on top of XBB.1.9.2:  C1263T(orf1a:T333M), A7062G(orf1a:Y2266C), C8692T, T13668G, G22801T
GISAID query: C8692T, T13668G, G22801T

There're only 3 XBB.1.9.2 seqs uploaded from China so far, and the 3 belong to the same branch. 

[usher](https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice9_genome_43448_3e4340.json)
<img width="1393" alt="Screen Shot 2023-03-17 at 11 57 50" src="https://user-images.githubusercontent.com/10445944/225810394-fcd78573-f1b5-4976-a314-8fc657475fc3.png">

orf1a:T333M is in
#535 BA.2.19
#237 AY.2.31.1

**Comment by c19850727** [comment]
EPI_ISL_17197742 also has a spike G504S mutation, potentially a mab escaping change.

**Comment by FedeGueli** [comment]
@c19850727 i found S:G504S in another chinese sequence but belonging to BA.5.2.48

**Comment by aviczhl2** [comment]
2 more of it. C_AA006201(Liaoning) and C_AA009410(Zhejiang)

However I don't know why usher doesn't group these 2 together with the previous 3.

**Comment by AngieHinrichs** [comment]
If I got to usher.bio, select the full tree (14M sequences including GISAID) and paste in C_AA006201.1, here's the subtree that I get from the 2023-04-16 tree:

https://nextstrain.org/fetch/hgwdev.gi.ucsc.edu/~angie/pango-designation-1769.2023-04-16.json?branchLabel=nuc%20mutations&c=gt-nuc_1263&label=id:node_5856071

-- So C_AA006201 and C_AA009410 have been placed with those sequences now.  Were you perhaps using the public-sequence-only tree (6.9M sequences)?  That does not include the GISAID sequences.  So before the GenBase sequences were added to the tree, there were only GISAID sequences with all these mutations, so the branch could not be found in the public tree.  

Now that the GenBase sequences have been added to the tree, they are included in the public tree (because they are not from GISAID).  So going forward, similar sequences should be placed with C_AA006201 and C_AA009410 even in the public tree.
